### ISSUE TITLE
Use Nipoppy schema to validate proc status TSV for bagel derivatives command

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/bagel-cli#352]

**Comment by alyssadai** [comment]
Currently blocked by #180, due to Nipoppy schema being written in v2 (and we would have to use their model as a module, not jsonschema, due to the internal validation they have)

**Comment by surchs** [comment]
Tracked while we discuss how to do this

**Comment by surchs** [comment]
Decision: We will implement the following as part of a new issue: grab example imaging status file from a nipoppy maintained repo (e.g. catalog) and use it as a test target. Keep this example in sync with upstream nipoppy (e.g. submodule) so tests start breaking when we go out of sync
